### PR TITLE
Issue41

### PR DESCRIPTION
**Pull request by katyhuff**
This PR handles most of the concerns in issue #41. It additionally adds explanations to the revise/revise.tex document, which we can send as part of the author response to the revisions (along with a cover letter). The following questions by the reviewer were addressed.

- Are all equations solved on the same mesh? In this case, is this a strict requirement? Please clarify.
- The MSRE had Reynolds numbers very close to the laminar-turbulent transition. Did you use a laminar model? In case, did you encounter convergence problems ? Or did you include turbulence? More generally, is Moltres capable of turbulent simulations?
- The authors are required to use the international system of units (meters, Kelvin, etc).
- The authors should briefly describe what "MSRE design models" are. Otherwise it is very difficult for the readers to assess the accuracy of the model.
- The authors claim an unprecedented accuracy of their model? How the results of their model compare with previous results from other authors? The presented results do not seem sufficient to support such claim.

However, the following four remain, before 41 can be closed:
- The heat generation in the graphite is calculated as a fraction of that in the fuel. How exactly is this achieved considering that the two sources are spatially separated?
- It is not clear how the domain is discretized. Is there a radial discretization in graphite and channels? Was the mesh convergence checked? Please discuss.
- Is the heat generation in the hottest channel the same for the Moltres and the MSRE model?
- The differences between the MSRE and the Moltres model are significant and should be discussed in more details. In particular the difference between fast fluxes is suspicious.

I am not certain of the best way to respond to the answers above (which should be briefly answerable by @lindsayad).

### CODE REVIEW

**Comment by gridley** [comment]
I made a PR on your branch to address the first unresolved point @katyhuff, just in case you don't notice.

File: Moltres.tex [hunk 1]
```diff
@@ -493,10 +493,17 @@ \subsection{Three dimensional simulation capability}
...
+\cite{briggs_molten-salt_1964} in the hottest channel and adjacent graphite. 
+The \gls{ORNL} \gls{MSRE} design calculations, conducted in 
+1963-1964,  were 32-group calculations using legacy computing tools (GAM-I, MODRIC, and 
+EQUIPOSE, and THERMOS). Those calculations were conducted in two-dimensional R-Z 
+geometry (a cylinder with angular symmetry), with 20 spatial regions. Notably, 
+one limitation of the \gls{ORNL} model was the control rod thimbles that, due to 
+angular symmetry, were effectively a cylindrical shell of metal. 
+
+The profile shapes are in decent qualitative agreement with both Moltres and
+\gls{MSRE} calculations, showing a peak in graphite temperature before the
```
Comment (lindsayad) [inline Moltres.tex]: Maybe no comma here

File: revise/revise.tex [hunk 2]
```diff
@@ -342,7 +353,12 @@ \section*{Reviewer 1}
 
 \question The proposed scaling test is very limited. Proper scaling tests would require weak and strong scaling tests, an assessment on the impact of the size of problem per core, a separate assessment of intra-node and extra-node scaling, etc. In addition, information should be provided on the employed numerical algorithms and on the parallelization strategy. The authors should either extend significantly the section, or remove it completely.
 \begin{solution}
-        We are performed strong and weak scaling up to 768 cores separately for intra-node and extra-node cases. Parallelization implemented using LibMesh library (for automatical domain decomposition) and PetSc library which contains set of parallel solvers. Detailed information added in Section 4.3 introduction.
+        We performed strong and weak scaling up to 768 cores separately for 
+        intra-node and extra-node cases. Parallelization was implemented using 
+        LibMesh library (for automatical domain decomposition) and PetSc
```
Comment (lindsayad) [inline revise/revise.tex]: perhaps change to "the LibMesh library for automatic domain decomposition and the PetSc library for parallel solution of systems of non-linear equations."

File: revise/revise.tex [hunk 3]
```diff
@@ -342,7 +353,12 @@ \section*{Reviewer 1}
 
 \question The proposed scaling test is very limited. Proper scaling tests would require weak and strong scaling tests, an assessment on the impact of the size of problem per core, a separate assessment of intra-node and extra-node scaling, etc. In addition, information should be provided on the employed numerical algorithms and on the parallelization strategy. The authors should either extend significantly the section, or remove it completely.
 \begin{solution}
-        We are performed strong and weak scaling up to 768 cores separately for intra-node and extra-node cases. Parallelization implemented using LibMesh library (for automatical domain decomposition) and PetSc library which contains set of parallel solvers. Detailed information added in Section 4.3 introduction.
+        We performed strong and weak scaling up to 768 cores separately for 
+        intra-node and extra-node cases. Parallelization was implemented using 
+        LibMesh library (for automatical domain decomposition) and PetSc 
+        library which contains set of parallel solvers. Detailed information, 
+        including results of the scalind studies, has now been added in the
```
Comment (lindsayad) [inline revise/revise.tex]: scaling

**Review by lindsayad — CHANGES_REQUESTED** [review]

**Comment by katyhuff** [comment]
Thanks folks. Merging.

**Pull request merged**